### PR TITLE
docs(copyedits): FRMW-265-Copy-Edits-styleguide

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-265

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

**Copy Edits** - Tables styleguide

### CODE REVIEW

File: demo/styleguide/tables.html [hunk 1]
```diff
@@ -56,7 +56,6 @@ <h3 class="title" id="delete-x">Delete "X" action</h3>
         <ul class="list">
             <li><span class="msg-info">ENCORE-SPECIFIC:</span> For data tables involving objects that only require deleting (Examples: tables related to LBaaS connection nodes and Support Service user tables) there's a design pattern that replaces the cog with an X.</li>
```
Comment (bry4n) [inline demo/styleguide/tables.html]: While you are modifying this file, do you think you can update `<li>` element to use 80 characters per line?

**before**
```html
  <li><span class="msg-info">ENCORE-SPECIFIC:</span> For data tables involving objects that only require deleting (Examples: tables related to LBaaS connection nodes and Support Service user tables) there's a design pattern that replaces the cog with an X.</li>
```

**after**
```html
<li>
    <span class="msg-info">ENCORE-SPECIFIC:</span> For data tables involving
    objects that only require deleting (Examples: tables related to LBaaS
    connection nodes and Support Service user tables) there's a design pattern
    that replaces the cog with an X.
</li>
```

Comment (100stacks) [inline demo/styleguide/tables.html]: @bry4n does Sublime have a shortcut way of doing this while preserving original line numbers?  I am worried about the various JIRA **Copy Edits** stories having possible PR merge conflicts if I change the actual line numbers.

Comment (bry4n) [inline demo/styleguide/tables.html]: @100stacks Hmm... you are right -- I just realized we are editing the same file.  We better hold off the 80 characers per line rule until we are done with copy edits.

Comment (100stacks) [inline demo/styleguide/tables.html]: Okay @bry4n :+1:  I agree the lines are WAY too long.  Plus, some content may move around with the new Information Architecture design.

Comment (100stacks) [inline demo/styleguide/tables.html]: screen shot 2015-10-08 at 4 08 30 pm

Comment (100stacks) [inline demo/styleguide/tables.html]: screen shot 2015-10-08 at 4 14 19 pm

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
@100stacks, please add a latest screenshot. Design will want it to add their LGTM.

**Comment by 100stacks** [comment]
screen shot 2015-10-12 at 11 36 00 am

**Comment by 100stacks** [comment]
@CITguy just added the latest screenshot with the _Copy Edit_ applied.  :+1:

**Comment by ssalinas24** [comment]
LGTM
The third bullet was deleted as the ticket suggested. I don't see any other issues with this.

**Pull request merged**